packages/react-composites/src/composites/common/VideoEffectsPane.tsx

```diff
@@ -145,9 +143,9 @@ export const VideoEffectsPaneContent = (props: {
   return VideoEffectsPaneTrampoline(
```

Comment:
We should rework the trampolines now if they are not needed

Reply:
Or in this case rename them to a more appropriate name as we associate trampolines with cced code.